connect/runtime/src/main/java/org/apache/kafka/connect/util/TopicAdmin.java

```diff
@@ -236,21 +236,21 @@ public boolean createTopic(NewTopic topic) {
                     continue;
                 }
                 if (cause instanceof UnsupportedVersionException) {
-                    log.debug("Unable to create topic(s) '{}' since the brokers at {} do not support the CreateTopics API.",
+                    log.debug("Unable to create topic(s) '{}' since the brokers at {} do not support the CreateTopics API." +
                             " Falling back to assume topic(s) exist or will be auto-created by the broker.",
                             topicNameList, bootstrapServers);
                     return Collections.emptySet();
                 }
                 if (cause instanceof ClusterAuthorizationException) {
                     log.debug("Not authorized to create topic(s) '{}'." +
                             " Falling back to assume topic(s) exist or will be auto-created by the broker.",
-                            topicNameList, bootstrapServers);
+                            topicNameList);
```

Comment:
Should we include the brokers instead of removing them? Same below.

Reply:
sure